{{cookiecutter.repo_name}}/requirements/base.txt

```diff
@@ -0,0 +1,6 @@
+django>=1.8.4,<1.9
+django-extensions
```

Comment:
this should pin at 1.5.5 at least as of right now; 1.5.6 has a [bug](https://github.com/django-extensions/django-extensions/issues/733) which affects our use case (TimeStampedModel).